### DISCUSSION TITLE
scikit-rf Merchandise

### DISCUSSION THREAD

**Discussion by jhillairet** [scikit-rf/scikit-rf#691] [Polls]
I was recently promoting scikit-rf in a domestic conference, and some colleagues suggest me to bring goodies/merchandise stuffs to popularize the package further (ideally, in a booth during conferences!).

What would like to purchase if you could? (unfortunately only single answer accepted, but please comment below your preferences)

**Comment by arsenovic** [comment]
i think this is great idea. i have some t-shirt design, that i can find . 

mug would be cool too

**Reply by jhillairet** [reply]
There are some online vendors that propose to easily generate T-shirt, mugs, etc, providing some logos. Unless someone wants to deal with buying/selling/shipping, I suggest using one of these platforms.

**Reply by arsenovic** [reply]
i agree. if you know of one you want to use ,i would be happy to get it setup

**Reply by jhillairet** [reply]
I've used once https://www.zazzle.fr  but I don't know if it can deliver all around the world. There are Python stuffs on https://www.redbubble.com and https://www.teepublic.com and https://www.freewear.org/Python too... I didn't check the various prices and options.

**Reply by jhillairet** [reply]
- Tshirts are between 19 and 22 euros or dollars on redbubble and teepublic, slightly more on zazzle
- Stickers seem to be of arbitrary shape on redbubble and teepublic, while only of predefined shape on zazzle
In all cases, the main advandtage of these platform is that you have choice of the product, shape, color, etc... 

overview of what it could be:
![image](https://user-images.githubusercontent.com/4642848/174436413-4d246e72-4b31-43e4-ad02-4aae2b15b009.png)
![image](https://user-images.githubusercontent.com/4642848/174436446-b915e626-2442-4d7a-8147-3e691e6e188c.png)

So it's time to be creative ;)

**Reply by jhillairet** [reply]
@arsenovic I've added dark background versions of the logos in #716, for neat dark background stuffs. Also added a simplified version of a Smith chart, for the ones who eventually want to wear it (It is also very nice in a clock!)

**Reply by arsenovic** [reply]
cool. i open an account at https://merchize.com/   seemed like tshirts were <10$.    i will see if i can add you onto the account. i like all these merch

**Reply by arsenovic** [reply]
also we have to keep the 'blam' on the shirts..... an important feature in my opinion. haha

**Comment by jhillairet** [comment]
A first merch is available at: 
http://scikit-rf.org/merch.html

feedback on designs/products/ordering are welcome.

**Comment by jhillairet** [comment]
Cool! I received the stickers! :-D

![image](https://user-images.githubusercontent.com/4642848/188805758-a31273c0-4f94-4c11-be2c-55d037c9acb4.png)

**Comment by arsenovic** [comment]
[image: 2022-09-07 10.48.39.jpg]

**Reply by jhillairet** [reply]
> [image: 2022-09-07 10.48.39.jpg]

I think this is not what you wanted to post, isn't it?
